modules/formatter/src/writer/ShapeWriter.scala

```diff
@@ -285,7 +285,7 @@ object ShapeWriter {
           members
         ) =>
       val content =
-        if (members.members.isEmpty) "{}"
+        if (members.members.isEmpty && !Comment.hasComment(members.ws0)) "{}"
```

Comment:
**question:** is this also handled in anonymous inputs/outputs?

Reply:
line 186 would confirm this:
```
      val content =
        if (members.members.isEmpty && !Comment.hasComment(members.ws0))
          "{}"
        else {
          val indented = indent(members.write, "\n", 4)
          s"{\n${indented}\n}"
        }
```

Reply:
oh great, thanks